build-logic/build-platform/build.gradle.kts

```diff
@@ -50,6 +52,7 @@ dependencies {
         api("org.jetbrains.kotlinx:kotlinx-metadata-jvm:0.1.0")
         api("org.jsoup:jsoup:1.11.3")
         api("org.junit.jupiter:junit-jupiter:5.7.0")
+        api("org.junit.vintage:junit-vintage-engine:5.7.0")
```

Comment:
We actually don't need this entry, because JUnit5 now does version alignment though it's BOM. It's enough to have the "junit-jupiter:5.7.0" entry.

Reply:
Doesn't seem to work without this https://ge.gradle.org/s/zz6mxa5bgsw5s

Reply:
My bad. I missed that we also use only the engine as dependency.